pkg/project/project.go

```diff
@@ -56,74 +39,19 @@ func Create(client *occlient.Client, projectName string, wait bool) error {
 
 // Delete deletes the project with name projectName and sets any other remaining project as the current project
 // and returns the current project or "" if no current project is set and errors if any
```

Comment:
outdated comment, and any reasons we removed the workflow of `sets any other remaining project as the current project`?

Reply:
Ack Fixed it now